src/avfoundation.cs

```diff
@@ -10696,7 +10803,6 @@ interface AVCaptureDevice {
 		[return: NullAllowed]
 		NSData GetExtrinsicMatrix (AVCaptureDevice fromDevice, AVCaptureDevice toDevice);
 
-		[Unavailable (PlatformName.MacCatalyst)]
 		[NoWatch, NoTV, NoMac, iOS (13,0)]
 		[Advice ("This API is not available when using UIKit on macOS.")]
```

Comment:
if you remove `[Unavailable (PlatformName.MacCatalyst)]`
then you should also remove the `[Advice ("This API is not available when using UIKit on macOS.")]` too